rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md

```diff
@@ -24,30 +24,36 @@ For everyone:
 
 * Less time stuck on a problem - you have someone familiar with the project to help you work through the problem
 * You can keep each other motivated  and you absorb knowledge from each other
-* Experience shows that **people** are more productive
+* Experience shows that **people** are more productive than any single person
```

Comment:
This wording is confusing. I don't know I would reword it though.

Reply:
Agreed a little confusing - how about...
Experience shows that two people working on a problem together are often more productive than one person working alone for twice as long.

Reply:
I've added that line instead of the original. Much better 💪